DOCSP-29819: type regression issue fixed in 4.8.1 (#688)
mongodb/docs-node: source/whats-new.txt

What's New in 4.8
-----------------

.. important:: Upgrade from v4.8.0 to v4.8.1

>    Version 4.8.1 includes a fix to a type regression issue that was
>    introduced in v4.8.0. By upgrading to v4.8.1, you can specify ``_id``

Reviewer: Suggestion

Language tightening

```suggestion
   Version 4.8.1 fixes a type regression issue
   introduced in v4.8.0. By upgrading to v4.8.1, you can specify ``_id``
```
